### PR TITLE
[MIG][16.0] sale_force_invoiced

### PR DESCRIPTION
**Pull request by alexis-via**
2 small improvements:

- force_invoiced is now tracked in the chatter (important !)
- force_invoiced is invisible on quotations

### CODE REVIEW

**Comment by alexis-via** [comment]
Tests are green, cool !

File: sale_force_invoiced/tests/test_sale_force_invoiced.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            {
+                "name": name,
+                "categ_id": product_ctg.id,
+                "type": "service",
+                "invoice_policy": "order",
+            }
+        )
+        return product
+
+    def _create_invoice_from_sale(self, sale):
```
Comment (StefanRijnhart) [inline sale_force_invoiced/tests/test_sale_force_invoiced.py]: Looks like you can remove this method now!

File: sale_force_invoiced/model/sale_order.py [hunk 2]
```diff
@@ -13,12 +13,13 @@ class SaleOrder(models.Model):
...
+        tracking=20,
         copy=False,
     )
 
     @api.depends("force_invoiced")
-    def _get_invoice_status(self):
-        res = super(SaleOrder, self)._get_invoice_status()
+    def _compute_invoice_status(self):
+        res = super()._compute_invoice_status()
         for order in self.filtered(
```
Comment (rousseldenis) [inline sale_force_invoiced/model/sale_order.py]: @alexis-via  You can improve:

`self.filtered(lambda so: so.force_invoiced and so.state in ("sale", "done")).update({"invoice_status": "invoiced"})`

**Review by StefanRijnhart — APPROVED** [review]

**Review by DavidJForgeFlow — APPROVED** [review]

**Review by ferran-S73 — APPROVED** [review]
Functional review OK

**Review by amarcosg — APPROVED** [review]

File: sale_force_invoiced/model/sale_order.py (hunk 2 above)
Comment (RodrigoBM) [reply sale_force_invoiced/model/sale_order.py]: @alexis-via can you add it?

**Review by Highcooley — APPROVED** [review]
Functional review OK

File: sale_force_invoiced/model/sale_order.py (hunk 2 above)
Comment (amarcosg) [reply sale_force_invoiced/model/sale_order.py]: Everything looks fine @alexis-via. Can you add this change?

Thanks!

Comment (alexis-via) [reply sale_force_invoiced/model/sale_order.py]: done

**Review by RodrigoBM — APPROVED** [review]

**Pull request merged**